urdfenvs/urdfCommon/urdfEnv.py

```diff
@@ -96,9 +115,13 @@ def _get_ob(self):
         return observation
 
     def addObstacle(self, obst):
+        # todo: refresh the observation space
         self._obsts.append(obst)
         obst.add2Bullet(p)
 
+    def getObstacles(self):
+        return self._obsts
+
```

Comment:
Where do you need this new function?

Reply:
In comparing the objects state with the object sensor. Thus for the unit test.